functional-tests/src/test/groovy/specs/HomeSpec.groovy

```diff
@@ -61,7 +58,7 @@ class HomeSpec extends GebReportingSpec {
       "pageTitle"                || HomePage
       "viewListBtn"              || ProjectListPage
       "viewMapBtn"               || ProjectMapPage
-      //TODO requires admin site to be running "activitiesAndUpdatesLink" || NewsPage
+      "activitiesAndUpdatesLink" || NewsPage
```

Comment:
do you wanna check if there are other commented out tests due to the admin site requirement? Since we are pointing to dev now we can enable them all again.

Reply:
Yeah, I had a look and I couldn't find any other commented-out tests.

Reply:
coolio